spec/controllers/users/webauthn_setup_controller_spec.rb

```diff
@@ -169,7 +169,7 @@
       controller.user_session[:webauthn_challenge] = webauthn_challenge
     end
     context ' Multiple MFA options turned on' do
-      let(:mfa_selections) { ['webauthn_platform', 'voice'] }
+      let(:mfa_selections) { ['webauthn', 'voice'] }
 
```

Comment:
Being new to Ruby I am struggling with what this syntax on lines 172 and 173 does. I am having trouble finding a good explanation online. Would someone mind explaining here?

Reply:
RSpec is a ruby library for testing. `let` is used to define lazily loaded, memoized, helper methods. The body of the method here is the block (the curly braces). When called, the method will return a 2-item array of strings